editor/src/components/canvas/canvas-strategies/post-action-options/navigator-reparent.ts

```diff
@@ -91,10 +92,35 @@ function getNavigatorReparentCommands(
       elementPath: path,
       pathToReparent: pathToReparent(path),
       intendedCoordinates: intendedCoordinates,
-      uid: EP.toUid(path),
+      newUID: EP.toUid(path),
     }
   })
 
+  const oldPathToNewPathMapping: OldPathToNewPathMapping = data.dragSources.reduce(
+    (acc: OldPathToNewPathMapping, value) => {
+      const childrenPaths = MetadataUtils.getChildrenPathsOrdered(
+        editor.jsxMetadata,
+        editor.elementPathTree,
+        value,
+      )
+
+      const pathAfterReparent = elementPathFromInsertionPath(newParentPath, EP.toUid(value))
+
+      const mapping: OldPathToNewPathMapping = [value, ...childrenPaths].reduce(
+        (accI: OldPathToNewPathMapping, valI) => ({
```

Comment:
Might be a bit clearer to read:
```suggestion
        (workingMappings: OldPathToNewPathMapping, path) => ({
```

Reply:
[0f76da3](https://github.com/concrete-utopia/utopia/pull/3890/commits/0f76da3a6a5188a7f08856845393ba804c6d4786) refactors this code, using less cryptic variable names